example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -152,13 +156,26 @@ describe('The S3 Ingest Granules workflow', () => {
     });
   });
 
-  describe('the sf-sns-report task has published a sns message and', () => {
-    it('the granule record is added to DynamoDB', async () => {
+  describe('An SNS success message', () => {
+    let lambdaOutput;
+    let existCheck;
+
+    beforeAll(async () => {
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PostToCmr');
+      executionName = lambdaOutput.cumulus_meta.execution_name;
+      existCheck = await s3ObjectExists({ Bucket: config.bucket, Key: `${config.stackName}/test-output/${executionName}.output` });
+    });
+
+    it('is published on workflow completion', async () => {
```

Comment:
I think it got lost in the shuffle of creating all these sub-tasks, but I used wording for acceptance criteria that could be translated more or less directly into integration test verbiage.  In this case:

"When Cumulus is configured to send a notification on successful workflow completion and an execution succeeds, it sends a notification to the configured SNS topic"
"When Cumulus is configured to send a notification on failed workflow executions and an execution fails, it sends a notification to the configured SNS topic"